spec/PlayerVsPlayerGameSpec.js

```diff
@@ -1,6 +1,31 @@
-var MockUI = require('./mocks/MockUI');
+var $ = require('jquery');
+var GameBoard = require('../src/GameBoard');
 var PlayerVsPlayerGame = require('../src/games/PlayerVsPlayerGame');
+var UI = require('../src/ui');
+var WinConditions = require('../src/WinConditions');
 
 describe("PlayerVsPlayerGame", function() {
-  
+
+  beforeEach(function() {
+    gameBoard = new GameBoard();
+    ui = new UI();
+    winConditions = new WinConditions();
+    game = new PlayerVsPlayerGame(ui, gameBoard, winConditions);
+    jasmine.getFixtures().fixturesPath = '../partials/'
+    jasmine.getFixtures().load('game.html');
+  });
+
+  describe("#play", function(){
+    it("will display a marker after a location has been chosen", function() {
```

Comment:
This does not look like it actually tests that the second move is alternating the piece. It looks like running the same test twice on two different spaces. That's not what we are looking for. Your game should be initialized once, and you should be making two moves on the same instance